tests/TESTFPC2.MLC

```diff
@@ -91,19 +95,20 @@ WTOPRC_END   EQU   *
                ELSE
                    MVC DRC,=X'40202020'
                    ED  DRC,PRC+6
-                   WTO MF=(E,WTOPRC)
-                   ABEND 102
+                   PUT RT1OUT,WTOPRC                                     #540
+                   RT1 ABORT           Close file and abend              #540
```

Comment:
Ends with ABEND 101 now, not ABEND 102. Same abend as previoius IF. Just pointing it out. See similar commnet in TESTFPC1.MLC changes.

Reply:
Same as above.